tee/tee_mock.go

```diff
@@ -118,3 +120,89 @@ func Decrypt(ct *tfhe.TfheCiphertext) (TeePlaintext, error) {
 
 	return plaintext, nil
 }
+
+func GetTeeCiphertextSize(t tfhe.FheUintType) (size uint, found bool) {
```

Comment:
@lovenoble Are you 100% sure that if I do:

```
tee.Encrypt(NewTeePlaintext(1, tfhe.Uint64))
tee.Encrypt(NewTeePlaintext(2, tfhe.Uint64))
```

the two ciphertexts will have the same length? Does that hold true for all elliptic curve signatures?

---

another question:

```
tee.Encrypt(NewTeePlaintext(1, tfhe.Uint64))
tee.Encrypt(NewTeePlaintext(1, tfhe.Uint32))
```

Are the 2 ciphertext lengths still the same?

Reply:
In general, I'd like to see in this PR the `teeCiphertextSize` table, what sizes we have.

For now we chose ECIES. This might change though, I would like to know if this table is still useful if we change to something else.

Reply:
The result will have the same length as long as the input length and the type are the same.
https://github.com/Inco-fhevm/zbc-go-ethereum/blob/master/crypto/ecies/ecies.go#L232
The output lenght is `ct = make([]byte, len(Rb)+len(em)+len(d))`, where 
```
	em, err := symEncrypt(rand, params, Ke, m)
	d := messageTag(params.Hash, Km, em, s2)
	Rb := elliptic.Marshal(pub.Curve, R.PublicKey.X, R.PublicKey.Y)
```
When we look into these functions, we can get to know that they will return the constant length for the input of same lenght.

As for the second question, it will usually have different lengths.